docs/semgrep-code/triage-remediation.md

```diff
@@ -149,112 +149,26 @@ To **reopen multiple findings** in the **No grouping** view, follow these steps:
-- You have completed a [Semgrep core deployment](/deployment/core-deployment). Particularly, you must have a successful connection to Bitbucket.
-
-### Enable triage through Bitbucket PR comments
-
-1. Sign in to Semgrep, and go to your organization's [Settings > General > Code](https://semgrep.dev/orgs/-/settings/general/code) page.
-2. Click the **Triage via code review comments** <i class="fa-solid fa-toggle-large-on"></i> toggle to turn on this feature.
-
-### Triage a finding
-
-1. Find an open comment created by Semgrep AppSec Platform in your pull request:
-    ![Semgrep AppSec Platform PR comment in Bitbucket](/img/bb-triage-by-pr.png#md-width)
+1. Find an open comment created by Semgrep in your pull request or merge request.
 2. In a subsequent comment, reply with the action you want to take. You must provide a reason to help the reader understand why the finding has been triaged as ignored:
     <TriageReason />
+    Semgrep attempts to reply to your comment if it successfully triages the finding:
+    ![Sample Semgrep response comment after successfully triaging the finding.](/img/semgrep-triage-response.png#md-width)
+    _**Figure**. Semgrep response comment after successfully triaging the finding._
 
-Triaging a finding as **Ignored** through a comment in Bitbucket changes the status of the finding to **Ignored** in Semgrep AppSec Platform. However, the Bitbucket conversation itself is **not** automatically resolved by this process.
-
-</TabItem>
+Triaging a finding as **Ignored** through a comment changes the status of the finding to **Ignored** in Semgrep AppSec Platform. However, the pull request or merge request conversation itself is **not** automatically resolved by this process.
 
-<TabItem value='gh'>
-
-### Prerequisites
-- A **private** GitHub Free or Team cloud-hosted repository or GitHub Enterprise repository. This feature is not enabled for public GitHub Cloud repositories.
-- You have completed a [Semgrep core deployment](/deployment/core-deployment).
-
-### Enable triage through GitHub PR comments:
-
-1. Sign in to Semgrep, and go to your organization's [Settings > General > Code](https://semgrep.dev/orgs/-/settings/general/code) page.
-2. Click the **Triage via code review comments** <i class="fa-solid fa-toggle-large-on"></i> toggle to turn on this feature.
-
-### Triage a finding
-
-1. Find an open comment created by Semgrep AppSec Platform in your pull request or merge request:
-    ![Screenshot of Semgrep AppSec Platform comment in GitHub](/img/semgrep-app-comment-github-beta.png#md-width)
-2. In a subsequent comment, reply with the action you want to take. You must provide a **reason** to help the reader understand why the finding has been triaged as **ignored**:
-    <TriageReason />
-
-Semgrep supports older versions of this functionality that used the following commands:
+:::info GitHub legacy commends
+For GitHub users, Semgrep supports older versions of this functionality that used the following commands:
```

Comment:
I don't think this is GitHub specific, AFAIK we just support these generally for backwards compatibility. Also there's a typo "commends".